offchain-modules/packages/app-multisign-server/src/ckbSigner.ts

```diff
@@ -235,6 +316,63 @@ async function verifyEthMintRecords(records: mintRecord[]): Promise<SigError> {
   return SigErrorOk;
 }
 
+async function verifyEthForUnlockRecords(records: unlockRecord[]): Promise<SigError> {
+  const unlockIds = records.map((record) => {
+    return record.id;
+  });
+  const ckbUnlocks = await SigServer.ckbDb.getCkbUnlockByIds(unlockIds);
+  const ckbUnlockMap: { [k: string]: CkbUnlock } = Object.fromEntries(
+    ckbUnlocks.map((ckbUnlock) => [ckbUnlock.id, ckbUnlock]),
+  );
```

Comment:
Should remove this, can not getCkbUnlockByIds when verify unlock(not send unlock tx yet). Verify unlockRecords match burnRecordsFromDb

Reply:
fixed, removed get unlock from db, get burnTxHash via splite '-' from id